esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -136,6 +137,8 @@ bool doAction(IEspContext& context, StringArray& wuids, CECLWUActions action, IP
                 {
                 case CECLWUActions_Restore:
                 {
+                    ensureWsWorkunitAccess(context, wuid, SecAccess_Full);
```

Comment:
Why was the required access changed from SecAccess_Write to SecAccess_Full.

Reply:
It is because it does not change the content of the WU.